github-actions/commit-message-based-labels/lib/main.ts

```diff
@@ -11,66 +12,107 @@ const supportedLabels = [
   [deprecationLabel, 'deprecations'],
 ] as const;
 
-async function main() {
-  let installationClient: Octokit | null = null;
+/** Label for docs changes. */
+const compDocsLabel = 'comp: docs';
 
-  try {
+class CommitMessageBasedLabelManager {
+  /** Run the commit message based labelling process. */
+  static run = async () => {
     const token = await getAuthTokenFor(ANGULAR_ROBOT);
-    installationClient = new Octokit({auth: token});
+    const git = new Octokit({auth: token});
+    const inst = new this(git);
+    await inst.run();
```

Comment:
If something fails in here, we don't revoke the token. That was the whole point of the "new pattern" of the `main` function that introduced in all actions. Any reason to diverge here?

Reply:
Ahh good, call, updated.